static/main.js

```diff
@@ -262,8 +262,9 @@ function initCodeMirror() {
         dummy.value = elem.doc.getValue()
         elem.setSize(
             null,
-            Math.max(dummy.scrollHeight - 5, elem.getTextArea().dataset.baseHeight || 27)
```

Comment:
does this mean the `dummy` element is no longer needed at all?

Reply:
No. It's still needed to set the other styles of the codemirror boxes.